html/index.ejs

```diff
@@ -19,34 +19,80 @@
         </p>
 
-        <div class="workflow-graphic"></div>
+        <div class="c-workflow-graphic">
+          <ul class="supply-chain-list">
+            <li class="supply-chain-item">
+              <span class="description -top paragraph -center">Where are they produced?</span>
+              <svg class="icon icon-supply icon-supply_chain_1"><use xlink:href="#icon-supply_chain_1"></use></svg>
+            </li>
+            <li class="supply-chain-item">
+              <svg class="icon icon-supply icon-supply_chain_2"><use xlink:href="#icon-supply_chain_2"></use></svg>
+              <span class="description -bottom paragraph -center">Who trades <br/>in them?</span>
+            </li>
+            <li class="supply-chain-item">
+              <span class="description -top paragraph -center">Where are they consumed?</span>
+              <svg class="icon icon-supply icon-supply_chain_3"><use xlink:href="#icon-supply_chain_3"></use></svg>
+            </li>
+            <li class="supply-chain-item">
+              <svg class="icon icon-supply icon-supply_chain_4"><use xlink:href="#icon-supply_chain_4"></use></svg>
+              <span class="description -bottom paragraph -center">What are the sustainability risks and opportunities?</span>
+            </li>
+            <li class="supply-chain-item">
+              <span class="description -top paragraph -center">And how can we <br/>make a difference?</span>
+              <svg class="icon icon-supply icon-supply_chain_5"><use xlink:href="#icon-supply_chain_5"></use></svg>
+            </li>
+          </ul>
+        </div>
 
-        <p class="paragraph -center">Transformative Transparency is a new kind of traceability platform.
+        <p class="paragraph -center"><span class="-bold">Transformative Transparency</span> is a new kind of traceability platform.
           It harnesses huge untapped data sets to see inside complex commodity supply chains,
           helping link production landscapes – and the social and environmental impacts
           and opportunities associated with them – to specific traders, manufacturers,
           retailers and investors worldwide.
         </p>
 
-        <a href="/flows.html" class="c-button -pink -bold">explore the tool</a>
+        <a href="/flows.html" class="c-button -big -pink -bold btn-explore">explore the tool</a>
 
-        <div class="map-selector">
-          <h3 class="intro">What are the sustainability risks associated with
-            the trading of
-            <select name="commodity" id="">Soy</select>
+        <div class="c-map-risks">
+          <h3 class="title -big -center -light">What are the sustainability risks associated with
+            the trading <br/>of
+            <div class="c-dropdown -homepage" data-dropdown="commodity">
+              <span class="js-dropdown-title dropdown-title">
+                -
+              </span>
+              <ul class="js-dropdown-list dropdown-list">
```

Comment:
Dropdowns overlap text which seems a bit weird. They should either show up below, or take the same space and be visually similar as the title text (check with JC)

![screen shot 2016-10-24 at 12 36 49](https://cloud.githubusercontent.com/assets/1583415/19642946/0f534106-99e8-11e6-8dfe-61e36ca88114.png)